store.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+</head>
+<body>
+    <header class="header">
+        <a href="#">
+            <img
+              src="img/karma-logo.svg"
+              alt="Larma Logo"
+              class="logo_header"
+            />
+        </a>
+  
+        <nav>
+            <ul class="navigation_list">
+              <li class="navigation_item">
+                <a class="navigation_link" href="#">Meet Karma</a>
```

Comment:
The Karma section of the nav bar is bold however the page is titled store.html.

Also the nav bar is not responsive to a hover effect in the same way that the index.html page is.